mirrord/layer/src/file/hooks.rs

```diff
@@ -985,6 +985,73 @@ unsafe extern "C" fn realpath_darwin_extsn_detour(
 //     result
 // }
 
+unsafe fn vec_to_iovec(bytes: &Vec<u8>, iovecs: &[iovec]) {
```

Comment:
This is one weird function signature. Can't this be `&[u8]` slice?

Reply:
Or maybe just pass the `Vec`?

Reply:
Also, do we need the whole thing to be `unsafe`? Isn't it only the pointer copying bits?

Reply:
I thought about passing `Vec` but that'd require keeping len and passing by ref is more gentle